### DISCUSSION TITLE
Navigate to Flutter screen

### DISCUSSION THREAD

**Discussion by dorinasimon99** [EnsembleUI/ensemble#391] [Q&A]
Hi!

Is there a way to navigate to a Flutter screen from an Ensemble screen?

**Comment by kmahmood74** [comment]
good question @dorinasimon99 There will be an enhancement required to enable navigating from ensemble to a flutter screen. Adding @vusters for comments. 

@dorinasimon99 could you please describe your use case here or in our discord - https://discord.gg/rxTQvftz

**Reply by dorinasimon99** [reply]
Use case:
- I have an existing Flutter app that I want to add Ensemble screens to
- From an Ensemble screen I want to navigate to a Flutter screen 

Example:

```
- Button:
          label: Navigate and pass inputs to a Flutter screen
          onTap:
            executeCode:
              body: |
                //@code
                ensemble.navigateFlutterScreen(
                  'MyFlutterWidget', # this is my screen's name
                  { # and here are the constructor parameters
                    'input': someInput 
                  }
                );
```

**Reply by kmahmood74** [reply]
Thanks @dorinasimon99 Just created an issue - https://github.com/EnsembleUI/ensemble/issues/440
Unable to tag you on it. Please follow it. We'll prioritize and get it done. 

Join our discord as well for active discussion - https://discord.gg/ASWswbqY

Please share your experience and ideas for improvement with Ensemble and let us know if you run into any issues. 

Thank you for the suggestion.

**Comment by kmahmood74** [comment]
hi @dorinasimon99 have a few mins to chat? can you describe the app you are building and your use case. We can chat on discord - https://discord.gg/ASWswbqY
